### PR TITLE
Redone the README

### PR DESCRIPTION
**Pull request by CleanMachine1**
I might make a few more edits if I see fit. I will tell you when I feel its ready to be reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
 ## Installing
-### Debian/Ubuntu
-check the latest release for debs
-### Other Linux
-compile yourself for now
-## compiling
+
+### To install for Debian/Ubuntu based distributions:
+
+To download for Debian/Ubuntu based distributions, simply go to Github [releases](https://github.com/Itai-Nelken/ish/releases) and install the package through dpkg or apt.
```
Comment (Itai-Nelken) [inline README.md]: For the link use https://github.com/Itai-Nelken/ish/releases/latest

That will always redirect to the latest release.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
+
+### To compile from source:
+
+This also works with Debian/Ubuntu based distributions however please use for distributions which don't use dpkg or apt.
```
Comment (Itai-Nelken) [inline README.md]: This line is unnecessary in my opinion.

**Review by Itai-Nelken — COMMENTED** [review]
Thanks a lot!
Just a few small things.

File: README.md (hunk 1 above)
Comment (CleanMachine1) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (CleanMachine1) [reply README.md]:
```suggestion
Compile from source when the `.deb` isn't useful for your distribution.
```

**Comment by CleanMachine1** [comment]
Right this is ready IMO after the edit that you have suggested is resolved

**Review by Itai-Nelken — APPROVED** [review]
Looks good, thanks!

**Pull request merged**